src/plugins/ImportKeras/ImportKeras.js

```diff
@@ -0,0 +1,307 @@
+        let config = layerConfig.config;
+        if (allPointerNames.length > 0) {
+            this.logger.debug(`Layer ${this.core.getAttribute(layerNode, 'name')}` +
+                ` has following configurable attributes ${allPointerNames.join(", ")}`);
+            allPointerNames.forEach((pointer) => {
+                // This case, there is no extra config for this attribute
+
+                if (config[pointer] && typeof config[pointer] === 'string') {
+                    let configurableNode = this.core.createNode({
+                        parent: layerNode,
+                        base: this.META[config[pointer]]
+                    });
+                    // This will set the necessary pointers.
+                    this.core.setPointer(layerNode, pointer, configurableNode);
+                    this.logger.debug(`Added ${this.core.getAttribute(configurableNode, 'name')}`
+                        + ` as ${pointer} to the layer `
+                        + `${this.core.getAttribute(layerNode, 'name')}`);
+                }
+                // The configuration will exist but inside the object.
+                else {
+                    if (!config[pointer]) {
+                    } else {
+                        let pluggableNode = this.core.createNode({
+                            parent: layerNode,
+                            base: this.META[config[pointer].class_name]
+                        });
+                        this.logger.debug(`Added ${this.core.getAttribute(pluggableNode, 'name')} as ${pointer} to the layer ` +
+                            `${this.core.getAttribute(layerNode, 'name')}`);
+                        let validArgumentsForThisNode = this.core.getValidAttributeNames(pluggableNode);
+                        let configForAddedNode = config[pointer].config;
+                        if (validArgumentsForThisNode && configForAddedNode) {
+                            validArgumentsForThisNode.forEach((arg) => {
+                                if (configForAddedNode[arg]) {
+                                    this.core.setAttribute(pluggableNode, arg, this._Stringify(configForAddedNode[arg]));
+                                }
+                            });
+                        }
+                    }
+                }
+            });
+        }
+    };
+
+    // This method is used to convert javascript arrays to a tuple/ list in string Representation.
+    ImportKeras.prototype._Stringify = function (obj) {
+        if (obj instanceof Array) {
+            return '[' + obj.map((val) => {
+                if (val) {
+                    return val;
+                } else {
+                    return "None";
+                }
+            }).join(', ') + ']';
+        } else {
+            return obj;
+        }
+    };
+
+    // This method is used to convert various classes from the keras JSON to deepforge meta Nodes
+    ImportKeras.prototype._mapClassName = function (orgName) {

```

Comment:
Changing the function name to something like `getMetaTypeForClass` and the input argument to `kerasClass` might make the comment more obvious.

Reply:
Something like this addresses the issue.
```javascript
ImportKeras.prototype._getMetaTypeForClass = function (kerasClass) {
        let classMap = ModelMaps.CLASS_MAP;
        if (Object.keys(classMap).indexOf(kerasClass) > -1) {
            return classMap[kerasClass];
        } else {
            return kerasClass;
        }
    };
```